lib/cryptoexchange/exchanges/cex/services/pairs.rb

```diff
@@ -0,0 +1,25 @@
+module Cryptoexchange::Exchanges
+  module Cex
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Cex::Market::API_URL}/currency_limits"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          pairs = output['data']['pairs']
+          pairs.map do |pair|
+            Cryptoexchange::Models::MarketPair.new(
```

Comment:
@rusbal 

From the API call https://cex.io/api/currency_limits, returns the following example response
```
{"pairs":[{"symbol1":"BTC","symbol2":"USD","minLotSize":0.01,"minLotSizeS2":2.5,"maxLotSize":30,"minPrice":"100","maxPrice":"15000"}

# symbol1 = BTC
# symbol2 = USD
```

The `base` should be mapped to `symbol1` because this is the currency that is being purchase
The `target` should be mapped to `symbol2` as its the currency used to purchase base

Reply:
`base` mapped to `symbol1`
`target` mapped to `symbol2`